test/TesterInternal/StatsTests.cs

```diff
@@ -10,31 +10,31 @@
 using TestExtensions;
 using Xunit;
 using Xunit.Abstractions;
+using Tester;
+using System.Collections;
 
 namespace UnitTests.Stats
 {
     public class StatsInitTests : OrleansTestingBase, IClassFixture<StatsInitTests.Fixture>
     {
         private readonly ITestOutputHelper output;
 
-        public class Fixture : BaseClusterFixture
+        public class Fixture : BaseTestClusterFixture
         {
-            protected override TestingSiloHost CreateClusterHost()
+            protected override TestCluster CreateTestCluster()
             {
-                return new TestingSiloHost(new TestingSiloOptions
-                {
-                    StartPrimary = true,
-                    StartSecondary = false,
-                    SiloConfigFile = new FileInfo("MockStats_ServerConfiguration.xml"),
-                    ReminderServiceType = GlobalConfiguration.ReminderServiceProviderType.ReminderTableGrain
-                }, new TestingClientOptions
-                {
-                    ClientConfigFile = new FileInfo("MockStats_ClientConfiguration.xml")
-                });
+                var options = new TestClusterOptions(initialSilosCount: 1);
+
+                options.ClusterConfiguration.Globals.RegisterStatisticsProvider<UnitTests.Stats.MockStatsSiloCollector>("MockStats");
+
+                options.ClientConfiguration.RegisterStatisticsProvider<UnitTests.Stats.MockStatsClientCollector>("MockStats");
+
```

Comment:
Do you need to set ReminderServiceType to GlobalConfiguration.ReminderServiceProviderType.ReminderTableGrain 
here or it is set to it in some other place?